### PR TITLE
Client Login showing any message when the api fail

### PR DESCRIPTION
**Pull request by NJavokhir**
### Issue:
The Client Login feature does not provide any feedback message to users when the API call fails, leading to a lack of clarity and frustration among users.

### Solution:
Implemented error handling to display informative messages to users when the API call for Client Login fails, enhancing user experience and facilitating troubleshooting.

### Changes Made:
- Added error handling mechanism to display relevant error messages when the Client Login API call fails.
- Updated UI to accommodate error message display without disrupting user flow.
- Resolved any conflicts.
- Tagged code owner and a second developer for review.
- Updated relevant documentation.

### Testing:
Ensured error message visibility through manual testing under various scenarios.

### Note:
Branch merging and deletion will be handled post-approval. Feedback for further refinement is appreciated.

### CODE REVIEW

**Review by ternakkode — CHANGES_REQUESTED** [review]
few comments

File: server/assets-v1/templates/src/pages/client_portal/login.html [hunk 1]
```diff
@@ -163,22 +162,31 @@ <h1 class="font-bold text-4xl text-[#3751FE] mb-11 animate-slideFromLeft">Login<
...
           );
           const responseTwo = await respTwo.json();
-          console.log("RESPONSE", responseTwo)
           if (responseTwo.token) {
             localStorage.setItem("clientToken", responseTwo.token);
-            alert("Login successful!");
+            showToastMessage("Login successful!", "success");
             window.location.href = "[[ .ProxyURL ]]/client-profile";
+          } else {
+            showToastMessage(responseTwo.message, "error");
```
Comment (ternakkode) [inline server/assets-v1/templates/src/pages/client_portal/login.html:170]: what if responseTwo doesnt have message field? i.e. 500 error

Comment (NJavokhir) [reply server/assets-v1/templates/src/pages/client_portal/login.html:170]: What do you think If I add extra 
```
else if (responseTwo.message) {
   showToastMessage(responseTwo.message, "error");
} else {
   showToastMessage("Something went wrong!", "error");
}
```

Comment (ternakkode) [reply server/assets-v1/templates/src/pages/client_portal/login.html:170]: LGTM

**Review by ternakkode — APPROVED** [review]
LGTM

**Pull request merged**